geometry_msgs/Point upleftPoint
geometry_msgs/Point downrightPoint
geometry_msgs/Point center
bool valid
